src/GuiBase/Viewer/TrackballCamera.cpp

```diff
@@ -202,14 +268,15 @@ void Gui::TrackballCamera::fitScene( const Core::Aabb& aabb ) {
     const Scalar y = r / std::sin( f * a / 2.0 );
     Scalar d = std::max( std::max( x, y ), Scalar( 0.001 ) );
 
-    m_camera->setPosition( Core::Vector3( aabb.center().x(), aabb.center().y(), aabb.center().z()+d ) );
+    m_camera->setPosition(
+        Core::Vector3( aabb.center().x(), aabb.center().y(), aabb.center().z() + d ) );
     m_camera->setDirection( Core::Vector3( 0, 0, -1 ) );
     m_trackballCenter = aabb.center();
 
     updatePhiTheta();
 
     m_distFromCenter = d;
-    m_cameraRadius = d;
+    m_distFromCenter = d;
```

Comment:
Duplicated line ?

Reply:
yes, these 2 attributes were exactly the same, missed the correction

Reply:
I think so ...